src/shared/pages/team/teamView/teamView.tsx

```diff
@@ -126,7 +129,7 @@ const TeamView: React.FC<TeamMembersProps> = ({ history, match, hasOneOfRoles })
             <div>
                 <h1 className="govuk-heading-xl">Manage Team</h1>
                 <h2 className="govuk-heading-l">
-                    {`Team: ${state.teamName}`}
+                    {`Team: ${state.teamName}`} ({state.active ? 'Active' : 'Inactive'})
```

Comment:
This is slight misuse of the h2, I think.

It might be nicer to explain to the user what "inactive" means, maybe via:
![image](https://user-images.githubusercontent.com/817054/122742649-13830d00-d27e-11eb-80bf-100d4c9d7b64.png)

I'd also only tell the user if the team was inactive: most teams they'll be dealing with are active teams, and we don't need to tell them if things are normal.

Consider using a tag: https://design-system.service.gov.uk/components/tag/

Reply:
I have addressed this.